content/backend/graphql-scala/0-introduction.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Introduction
+pageTitle: "Building a GraphQL Server with Scala Backend Tutorial"
+description: "Learn how to build a GraphQL server with Scala & Sangria and best practices for filters, authentication, pagination and subscriptions. "
+
+---
+
+### Motivation
+
+Scala is very popular language nowadays and it's often chosen to deliver efficient and distributed systems. It leverages the Java VM, known mostly for its efficiency. Support of Functional Programming and power of Java make able to deliver applications fast in either development or runtime.
+
+In the next chapters you'll learn how to build your own GraphQL server using Scala and the following technologies:
+  * [Scala](https://www.scala-lang.org/) Scala language
+  * [Akka HTTP](http://doc.akka.io/docs/akka-http/current/scala/http) Web server to handle HTTP requests.
+  * [Sangria](http://sangria-graphql.org/) A library for GraphQL execution
+  * [Slick](http://slick.lightbend.com/) A Database query and access library.
+  * [H2 Database](http://www.h2database.com/html/main.html) In-memory database.
+  * [Graphiql](https://github.com/graphql/graphiql) A simple GraphQL console to play with.
+
+I assume you're familiar with GraphQL concepts, but if not, you can visit [GraphQL site](http://graphql.org/) to learn more about that.
+
+### What is a GraphQL Server?
+
+A GraphQL server should be able to:
+
+* Receive requests following the GraphQL format, for example:
+
+```graphql(nocopy)
+{  "query": "query { allLinks { url } }" }
+```
+
+* Connect to any necessary databases or other data management services.
```

Comment:
... to one or more data sources, like data bases or other APIs, obtain the information and format it